test/export-variables/fixtures/class.esm.default.js

```diff
@@ -1 +1 @@
-class a{constructor(b){this.name_=b}console(){console.log(this.name_)}}export{a as Exported};
+class a{constructor(b){this.name_=b}console(){console.log(this.name_)}}var Exported=a;export{Exported};
```

Comment:
This is an interesting example of how the new approach eliminates a large number of conditions the previous code was accounting for.

It's unfortunate we are creating the extra reference just to export it.